app/models/merchant.rb

```diff
@@ -9,12 +12,22 @@ def favorite_customers_alt
     customers.top_5_customers_by_transaction(self.id)
   end
   
+  def self.find_by_status(merchant_status)
+    Merchant.where(status: merchant_status).order(updated_at: :desc)
```

Comment:
What's the thought process behind sorting merchants by when they were last updated?

Also, what's the difference between this method and `self.enabled_merchants` and `self.disabled_merchants`?